handbook/company/README.md

```diff
@@ -58,13 +58,13 @@ Empathy leads to smarter decisions.  Take an interest in what people are going t
 - **Assume positive intent.** Think and say [positive things](https://www.theatlantic.com/family/archive/2018/06/mr-rogers-neighborhood-talking-to-kids/562352/), and [assume](https://about.gitlab.com/handbook/values/#assume-positive-intent) others are doing the same.
 - **Be a helper.** Take care of customers first. But give hospitality and [service with a smile](https://en.m.wikipedia.org/wiki/Fred_Rogers#Legacy) to everyone you can.
 - **Roleplay.** Read what you write.  [Again.](http://www.paulgraham.com/useful.html)  Use your imagination to see situations from different perspectives.
-- **Get curious.** Genuinely care. Ask questions. Listen closely to the answers. Be present.
+- **Get curious.**  Genuinely wonder. Ask questions. Listen closely to the answers.
 
 ### 🟠 Ownership
-It takes reliable execution and a fully-activated mind to achieve ambitious goals. Think like an owner of the company.
+It takes a fully-activated mind to achieve ambitious goals. Think like an owner of the company.
```

Comment:
reliable execution above was axed because it made it more unclear which value was value when I was testing it out with real-world examples